azurerm/internal/services/containers/resource_arm_kubernetes_cluster.go

```diff
@@ -1487,29 +1499,31 @@ func expandLoadBalancerProfile(d []interface{}, loadBalancerType string) (*conta
 
 	config := d[0].(map[string]interface{})
 
-	var managedOutboundIps *containerservice.ManagedClusterLoadBalancerProfileManagedOutboundIPs
-	var outboundIpPrefixes *containerservice.ManagedClusterLoadBalancerProfileOutboundIPPrefixes
-	var outboundIps *containerservice.ManagedClusterLoadBalancerProfileOutboundIPs
+	loadBalancerProfile := &containerservice.ManagedClusterLoadBalancerProfile{}
```

Comment:
for the sake of brevity could we use
```suggestion
	profile := &containerservice.ManagedClusterLoadBalancerProfile{}
```

Reply:
Updated